tests/Unit/ActionTesting.hpp

```diff
@@ -501,6 +501,14 @@ class MockArrayElementProxy {
     local_algorithm_.template simple_action<Action>();
   }
 
+  // This doesn't do anything, but it allows to test actions that set this flag.
+  bool terminate{false};
+  void set_terminate(bool t) noexcept { terminate = t; };
+
+  // Actions may call this, but since tests step through actions manually it has
+  // no effect.
+  void perform_algorithm() noexcept {};
```

Comment:
In the long-term this should have an observable effect, but I don't know what that should be just yet. https://github.com/sxs-collaboration/spectre/issues/1016